lib/customValidators.js

```diff
@@ -174,6 +174,70 @@ function validateOperationId(
   return true;
 }
 
+/**
+ * Validates if messageIds are duplicated in the document
+ *
+ * @private
+ * @param  {Object} parsedJSON parsed AsyncAPI document
+ * @param  {String} asyncapiYAMLorJSON AsyncAPI document in string
+ * @param  {String} initialFormat information of the document was originally JSON or YAML
+ * @returns {Boolean} true in case the document is valid, otherwise throws {@link ParserError}
+ */
+function validateMessageId(
```

Comment:
@magicmatatjahu I see your suggestions on this method.
This method is a (almost) copy of the already existing method above in this file,

This method is copy of already existing https://github.com/asyncapi/parser-js/blob/d3dd11d885074c29ef5cde8956c5968351a54d7a/lib/customValidators.js#L123

I tried to keep it consistent with that. For example this comment :D
> it is a map of paths, the one that is a duplicate and the one that is duplicated

Let me know if we still need to make changes.

Reply:
Ok 😄  Please go ahead and don't worry about it. We should fix it in the another PR.